src/integrations/stytch.js

```diff
@@ -129,12 +141,7 @@ function removeStytchQueryParams() {
   }
 }
 
-export function googleOauthRedirect() {
-  const stytchClient = getAuthClient();
-  if (!stytchClient) return;
-
-  stytchClient.oauth.google.start({
-    login_redirect_url: window.location.href,
-    signup_redirect_url: window.location.href
-  });
+function getRedirectUrl() {
+  const { origin, pathname, hash } = window.location;
+  return `${origin}${pathname}${hash}`;
```

Comment:
it would be fair to ask why I construct this URL instead of using URLSearchParams. It's just simpler to construct this rather than needing to iterate over URLSearchParams keys, delete the keys, and then get the URL